tox.ini

```diff
@@ -3,7 +3,8 @@ envlist =
     flake8
     py{36}-dj{20}-wt{23}
     py{36,37}-dj{21}-wt{24}
-    py{36,37,38}-dj{22}-wt{25,26,27,28,29}
+    py{36,37,38}-dj{22}-wt{25,26,27,28}
+    py{37,38}-dj{30}-wt{28,29}
```

Comment:
@lingster This line should be changed to: `py{37,38}-dj{30}-wt{28,29,**210**}`

Reply:
done, I think at the time of these changes wagtail 2.10.x had not been released